app/Services/Parser/Parsers/Link.php

```diff
@@ -163,7 +163,16 @@ protected function parseYoutubeLinks($text, $match, $url, $title)
             if ($host === 'youtu.be' && $path !== '') {
                 parse_str($components['query'] ?? '', $query);
 
-                $text = str_replace($match, $this->makeIframe($path, $this->timeToSeconds($query['t'] ?? null)), $text);
+                $text = str_replace(
+                    $match, 
+                    $this->makeIframe(
+                        $path, 
+                        (isSet($query['t']) && $this->timeToSeconds($query['t'])) 
```

Comment:
`isset()` zamiast `isSet()`

Poza tym wydaje mi sie ze tego kodu nie trzeba w ogole modyfikowac. Bo jak rozumiem wszystko rozchodzi sie o to, aby metoda `timeToSeconds` potrafila przekonwertowac format `XmXs` do postaci sekund, ale w przypadku gdy argumentem tej metody sa sekundy to nie robimy nic. Zgadza sie?

Reply:
Tutaj chodzi o to, że bez `isset()` będzie błąd w następnej linii w przypadku linku z youtu.be bez znacznika `t` - nie przechodzi test `tests/unit/LinkTest.php:testYoutubeVideos` - zwraca `[ErrorException] Undefined index: t` w `/var/www/tests/unit/LinkTest.php:135`,  czyli `$this->tester->assertContains('iframe', $parser->parse('https://youtu.be/enOjqwOE1ec'));`

Reply:
Eee, chyba nie. Ten Twój kod można uprościć do takiej postaci:

`$text = str_replace($match, $this->makeIframe($path, $this->timeToSeconds($query['t'] ?? null)), $text);`

Wszystko dzięki operatorowi `??`.

Reply:
no racja, w złym miejscu sobie postawiłem ten operator po którejś tam już zmianie